custom_components/powercalc/aliases.py

```diff
@@ -92,7 +92,27 @@
         "TS0505B": "NO66-ZB/length_5",
     },
     "Yeelight": {
+        "mono": "YLTD03YL",
+        "mono1": "YLDP01YL",
+        "color1": "YLDP02YL_YLDP03YL",
```

Comment:
These are multiple models for the same yeelight model. 
This is not going to work because it needs to map one to one.
There was also some discussion about the same topic here https://github.com/bramstroker/homeassistant-powercalc/pull/690
Also I don't like to add all these aliases in advance when they are not part of the library yet, to not pollute with unused stuff.

Reply:
Okay. I just saw the opportunity to add them all in one go. I'll comment out the unused once, to clean up the database - but avoid having to read the docs all the time you add a new one. Is that a good compromise? 🤔

Reply:
The issue was(/is) that we got some profiles which are currently not automatically identified, because the mapping is missing. :)